### ISSUE TITLE
SmartBlock command to lose focus after workflow completion - especially for buttons

### ISSUE THREAD

**Issue by eatondpe** [roamhacker/SmartBlocks#101]
## 📋 Describe the new feature/idea/enhancement you are requesting
Need a way to tell the SmartBlock engine to lose focus on any created/modified block at the end of a workflow.
## 🔑 Why do you want this?
Many times, especially with buttons, having the workflow complete with a block that is not intended to have more content added be in focus breaks the flow requiring an unnecessary mouse click or key sequence to lose or change the focus.
## ⚠️ Is your feature request related to a problem? Please describe.
When clicking a button, such as a counter, that produces only another button, that next button should be clickable without manually exiting focus, such as clicking outside the block.
## 🔨 Describe any alternatives you've considered
No known alternative, other than clicking outside the block after the workflow completes.
## 📷 Additional Info (Screenshots, GIFs, etc.)

---
[label_idea]

**Comment by GitMurf** [comment]
@eatondpe I totally agree! The only workaround I've found is adding a new empty block below it and putting cursor there. But that's not ideal. Basically you want to add a key press of "escape" key at the end right?
